screens/DetailsProductScreen.js

```diff
@@ -1,32 +1,61 @@
 import React from 'react';
-import { StyleSheet, Text, View, Button } from 'react-native';
+import { StyleSheet, Text, View, Button, Image } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import CustomButton from '../components/CustomButton';
```

Comment:
This should be after react packages in a import area, in a place where you import custom files and components.